go.mod

```diff
@@ -16,9 +16,9 @@ require (
 	github.com/fsnotify/fsnotify v1.7.0
 	github.com/go-ldap/ldap/v3 v3.4.6
 	github.com/gofrs/uuid v4.4.0+incompatible
-	github.com/google/go-containerregistry v0.16.1
+	github.com/google/go-containerregistry v0.17.0
 	github.com/google/uuid v1.4.0
-	github.com/gorilla/mux v1.8.0
+	github.com/gorilla/mux v1.8.1
```

Comment:
This one is broken. We already tried. See this comment: https://github.com/project-zot/zot/pull/2040#issuecomment-1810143357

Reply:
looks like we have to force it to that version